### DISCUSSION TITLE
Custom audio channel selection

### DISCUSSION THREAD

**Discussion by ttopac** [f1multiviewer/issue-tracker#261] [Feature requests] [answered]
Hi, and thanks for the awesome work!

Before F1MV, I've been watching the main stream and my favorite driver's OBC on two separate devices. In that configuration, I separated the commentator and team radio audio by using two headphones, one playing commentator and one playing team radio. 

I found that configuration quite satisfying, and I think F1MV should implement a feature to direct different audio sources to different audio channels. In the current configuration, it's not easy to understand when both team radio and commentator speaks at the same time.

**Comment by f1multiviewer** [comment, marked as answer]
You can actually already do this! Right click on the player to set the "Set Audio Device":
<img width="247" alt="image" src="https://user-images.githubusercontent.com/106821994/197524125-ba6db831-3af3-4f64-ace2-cb7e470ff273.png">

**Reply by ttopac** [reply]
Thanks! That's good to know and will certainly be helpful. I think this can get even more useful if we can have control over individual audio channels: One player sends audio to the left earbud, and the other one to the right one.

**Reply by ttopac** [reply]
Actually, it seems like I can get this functionality if I use [Loopback](https://rogueamoeba.com/loopback/), but it'd be good to have it built-in.

**Reply by f1multiviewer** [reply]
Yeah loopback is going to be the way, I can't do it for other streams (non-OBC) because it introduces latency, which is unwanted and was an issue before.
